plugin/src/App/init.lua

```diff
@@ -470,7 +523,11 @@ function App:render()
 					}),
 				}),
 
-				RojoNotifications = e("ScreenGui", {}, {
+				RojoNotifications = e("ScreenGui", {
+					ZIndexBehavior = Enum.ZIndexBehavior.Sibling,
+					ResetOnSpawn = false,
+					DisplayOrder = 100,
```

Comment:
Same question as with the LayoutOrders; what's the reason this is so large?

Reply:
Rojo Notifications need to display on top for users to see. We don't know what else may be in their CoreGui, so we just use a high display order to increase our odds of having notifications viewable.